pkg/scheduler/eventhandlers.go

```diff
@@ -362,6 +362,14 @@ func addAllEventHandlers(
 			informerFactory.Storage().V1().CSINodes().Informer().AddEventHandler(
 				buildEvtResHandler(at, framework.CSINode, "CSINode"),
 			)
+		case framework.CSIDriver:
+			informerFactory.Storage().V1().CSIDrivers().Informer().AddEventHandler(
+				buildEvtResHandler(at, framework.CSIDriver, "CSIDriver"),
+			)
+		case framework.CSIStorageCapacity:
+			informerFactory.Storage().V1beta1().CSIStorageCapacities().Informer().AddEventHandler(
```

Comment:
The CI failure may be due to this. I think it should be V1().

Reply:
`pull-kubernetes-e2e-gce-alpha-features` fails due to this issue https://github.com/kubernetes/kubernetes/issues/103440

`pull-kubernetes-e2e-gce-alpha-features` failure is unrelated, can be ignored in this PR